src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java

```diff
@@ -1,6 +1,7 @@
 package edu.kis.powp.jobs2d.command.gui;
 
 import edu.kis.powp.appbase.gui.WindowComponent;
+import edu.kis.powp.jobs2d.command.CommandPreviewPanelController;
```

Comment:
The code is extracted but dependencies still exist. Try to lower coupling, maybe introduce an interface and dependency injection.